tests/src/test/java/alluxio/client/fs/FileOutStreamAsyncWriteIntegrationTest.java

```diff
@@ -204,103 +218,55 @@ public void asyncPersistNoAutoPersistNoEvict() throws Exception {
    */
   private void testLostAsyncBlocks() throws Exception {
     long cap = FormatUtils.parseSpaceSize(TINY_WORKER_MEM);
-    FileSystem fs = mLocalAlluxioClusterResource.get().getClient();
-    // Create a large-ish file in the UFS (relative to memory size)
-    String p1 = "/test";
-    FileSystemTestUtils.createByteFile(fs, p1, WritePType.ASYNC_THROUGH, (int) cap);
-
-    BlockWorker blkWorker = mLocalAlluxioClusterResource.get().getWorkerProcess()
-        .getWorker(BlockWorker.class);
-    SpaceReserver reserver = Whitebox.getInternalState(blkWorker, SpaceReserver.class);
-
-    // Trigger worker eviction
-    reserver.heartbeat();
-    URIStatus fstat = fs.listStatus(new AlluxioURI(p1)).get(0);
-    int lostBlocks = fstat.getFileBlockInfos().stream()
-        .map(FileBlockInfo::getBlockInfo)
-        .filter(blk -> blk.getLocations().size() <= 0)
-        .mapToInt(blk -> 1)
-        .sum();
-
-    assertEquals(cap, getClusterCapacity());
-    assertEquals(cap, getUsedWorkerSpace());
-    assertEquals(100, fstat.getInAlluxioPercentage());
-    assertEquals(0, lostBlocks);
-
-    FileSystemUtils.persistAndWait(fs, new AlluxioURI(p1), 0);
-    fstat = fs.listStatus(new AlluxioURI(p1)).get(0);
-
-    assertTrue(fstat.isPersisted());
-    assertEquals(0, mLocalAlluxioClusterResource.get().getLocalAlluxioMaster().getMasterProcess()
-        .getMaster(FileSystemMaster.class)
-        .getPinIdList().size());
-    assertTrue(getUsedWorkerSpace() < getClusterCapacity());
+    try (FileSystem fs = sLocalAlluxioClusterResource.get().getClient()) {
```

Comment:
why not use `mFileSystem`?

Reply:
There are tests which set some user configuration beforehand. I don't use `mFileSystem` because I want the client used to pick up any changes to configuration properties